builtin/credential/ldap/path_config.go

```diff
@@ -143,8 +166,22 @@ func (b *backend) pathConfigWrite(
 	if groupdn != "" {
 		cfg.GroupDN = groupdn
 	}
+	groupfilter := d.Get("groupfilter").(string)
```

Comment:
OK, that sounds good then. Do think about/check on the legacy `memberOf` searching too. That was a totally separate block of logic and I don't see any corresponding handling in your code.